scripts/docs.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#####################################################
+# This script is used for Continuous Integration
+#
+# Run locally to verify before committing your code.
+#
+# Options:
+#   -g to generate documentations.
+#   -u to upload documentation to google cloud.
+#
+#####################################################
+
+set -o pipefail
+set -e
+
+# get platforms
+GENERATE=false
+UPLOAD=false
+
+while true; do
+  case "$1" in
+    -g  ) GENERATE=true;UPLOAD=false;;
```

Comment:
```suggestion
    -g  ) GENERATE=true;;
```
No reason generate and upload couldn't both be run in the same script execution, is there?

Reply:
Yes, you are right, we can generate and upload doc with the same script